upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -299,14 +299,14 @@ func (c *ApplyClusterCmd) Run() error {
 
 		if warn {
 			fmt.Println("")
-			fmt.Println(starline)
+			fmt.Printf(("%s\n"), starline)
```

Comment:
Why all the extra parentheses?

Reply:
This is my syntax fault .Thank you for pointing it out.